### ISSUE TITLE
El id de un campo es obligatorio

### ISSUE THREAD

**Issue by Gochip** [dsbarrionuevo/ValidationJS#12]
El id de un campo es obligatorio.

Esto es correcto? O habría que cambiarlo?

Por ejemplo, hay casos en que me quiero referir a varios campos por la clase y todos ellos no tienen un id asociado...

**Comment by dsbarrionuevo** [comment]
Sí, ahora el validator de un formulario mantiene un array asociativo donde la clave es el id del campo y el valor es un array de todas sus validaciones. Debería yo autoasignar los ids del array asociativo y poder hacer que al usar un selector más general como el de clase se pueda agregar una validación a muchos campos. También se relaciona con #11 .
